docs/group-user-management.rst

```diff
@@ -0,0 +1,227 @@
+Administrate Groups and Users
+=============================
+
+Description:
+------------
+
+This chapter will show how to manage groups and users using graphical interface in OMERO.web and command line interface. Most of following tasks below can only be done by users with some
+administrator privileges. We will show:
+
+- How to manage groups, creating and editing a new/existing group
+- How to manage users, creating and editing a new/existing user
+
+**Resources:**
+--------------
+
+-  Documentation:
+
+   -  https://docs.openmicroscopy.org/latest/omero/sysadmins/server-permissions.html
+
+   -  https://docs.openmicroscopy.org/latest/omero/sysadmins/restricted-admins.html
+
+   -  https://docs.openmicroscopy.org/omero/latest/sysadmins/cli/usergroup.html
+
+
+Setup:
+------
+
+No setup needed for OMERO.web administration panel (see Web Interface chapter below) except working OMERO.web.
+
+**Command Line interface installation**
+
+The installation instructions can be
+found at \ https://docs.openmicroscopy.org/latest/omero/users/cli/installation.html\ .
+
+
+**Step-by-step:**
+-----------------
+
+Administrate using the Web Interface
+------------------------------------
+
+Below are some useful links to know more about permissions and administrator privileges:
+
+-  https://docs.openmicroscopy.org/latest/omero/sysadmins/server-permissions.html
+
+-  https://docs.openmicroscopy.org/latest/omero/sysadmins/restricted-admins.html
```

Comment:
These links are repeated in the previous section. Bit confusing (user needs to check if they're the same links).

Reply:
Fixed in https://github.com/ome/omero-guide-introduction/pull/1/commits/b04121dae66a98ac383a0c70b5526ce4beb6fb94